### ISSUE TITLE
Esc and focus trap does not work if iframe is inside the modal

### ISSUE THREAD

**Issue by jacktan165** [ghosh/Micromodal#504]

**Comment by arek-kilimar-allhuman** [comment]
Is there any update on this. The problem still exists.

**Comment by kartmax** [comment]
I encountered the same problem - the modal window does not close by pressing the Esc button if there is an iframe inside it

**Comment by kartmax** [comment]
> I encountered the same problem - the modal window does not close by pressing the Esc button if there is an iframe inside it

I figured it out - you need to include this setting in the object when initializing the modal window - disableFocus : true,
After that, the Esc button worked

**Comment by rtkirill** [comment]
Same issue for me, when including iframe. Changing disableFocus didn't help.

**Comment by dkniffin** [comment]
@rtkirill @kartmax @arek-kilimar-allhuman @jacktan165 Can one of you create a codepen to reproduce this issue? Here's a template you can base it on: [Codepen example](https://codepen.io/pen?template=LEYmYWy)

**Comment by arek-kilimar-allhuman** [comment]
@dkniffin, if you add an <iframe> to the modal container, the focus moves out to the page once it passes the iframe element.

This happens only when the iframe is the last (or possibly the first) interactive element in the modal.
If you place a simple <a> element after the iframe, the focus remains trapped within the modal.
